app/models/gem_download.rb

```diff
@@ -33,6 +33,12 @@ def self.total_count
     end
   end
 
+  # version_id: 0 stores count for total gem downloads
+  # we need to find the second maximum
+  def self.most_downloaded_count
```

Comment:
Let's name this `most_downloaded_gem_count` to be clear this is for gems not versions.